src/media/Icon.jsx

```diff
@@ -50,7 +120,7 @@ class Icon extends React.PureComponent {
 }
 
 Icon.defaultProps = {
-	size: 'm'
+	size: 's'
 };
 
 Icon.propTypes = {
```

Comment:
So there is another ticket for this but look was talking to you about this @mperrotti, xxl was removed from accepted list but it's being used. However it's in the `MEDIA_SIZES` i suggest updating the sizes prop to look be

`size: React.PropTypes.oneOf(Object.keys(MEDIA_SIZES))`

then your not maintaining 2 lists :)